src/SaveHandler/MongoDB.php

```diff
@@ -10,62 +10,60 @@
 
 namespace Zend\Session\SaveHandler;
 
-use Mongo;
-use MongoClient;
-use MongoCollection;
-use MongoDate;
+use MongoDB\BSON\Binary;
+use MongoDB\BSON\UTCDatetime;
+use MongoDB\Client as MongoClient;
+use MongoDB\Collection as MongoCollection;
 use Zend\Session\Exception\InvalidArgumentException;
 
 /**
  * MongoDB session save handler
  */
-class MongoDB implements SaveHandlerInterface
+final class MongoDB implements SaveHandlerInterface
```

Comment:
Agreed with @Maks3w on this one. If this were a new class, final/private would be fine, but for existing classes, we need to keep visibility the same.

We should definitely revisit for v3.